http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/monitor/ZombieRuntimeMonitor.scala

```diff
@@ -0,0 +1,169 @@
+package org.broadinstitute.dsde.workbench.leonardo
+package monitor
+
+import java.time.Instant
+import java.util.UUID
+import java.util.concurrent.TimeUnit
+
+import cats.Parallel
+import cats.effect.concurrent.Semaphore
+import cats.effect.{Concurrent, ContextShift, IO, Timer}
+import cats.implicits._
+import cats.mtl.ApplicativeAsk
+import com.google.api.client.googleapis.json.GoogleJsonResponseException
+import fs2.Stream
+import io.chrisdavenport.log4cats.Logger
+import org.broadinstitute.dsde.workbench.google.GoogleProjectDAO
+import org.broadinstitute.dsde.workbench.google2.{GoogleComputeService, InstanceName}
+import org.broadinstitute.dsde.workbench.leonardo.config.ZombieClusterConfig
+import org.broadinstitute.dsde.workbench.leonardo.dao.google.GoogleDataprocDAO
+import org.broadinstitute.dsde.workbench.leonardo.db._
+import org.broadinstitute.dsde.workbench.leonardo.http._
+import org.broadinstitute.dsde.workbench.model.TraceId
+import org.broadinstitute.dsde.workbench.model.google.GoogleProject
+import org.broadinstitute.dsde.workbench.newrelic.NewRelicMetrics
+
+import scala.concurrent.ExecutionContext
+
+/**
+ * This monitor periodically sweeps the Leo database and checks for clusters which no longer exist in Google.
+ */
+class ZombieClusterMonitor[F[_]: Parallel: ContextShift: Timer](
+  config: ZombieClusterConfig,
+  gdDAO: GoogleDataprocDAO,
+  gce: GoogleComputeService[F],
+  googleProjectDAO: GoogleProjectDAO
+)(implicit F: Concurrent[F],
+  metrics: NewRelicMetrics[F],
+  logger: Logger[F],
+  dbRef: DbReference[F],
+  ec: ExecutionContext,
+  cs: ContextShift[IO]) {
+
+  val process: Stream[F, Unit] =
+    (Stream.sleep[F](config.zombieCheckPeriod) ++ Stream.eval(F.delay(println("evaluating")))++ Stream.eval(zombieCheck)).repeat
```

Comment:
maybe `Logger` instead of `println`?

Reply:
will remove...we don't need this..was there when I was setting up things in unit test